scrapers_next/ia/committees.py

```diff
@@ -0,0 +1,95 @@
+    def process_page(self):
+        # Helper method to parse member list item strings
+        def _parseMemberString(member: str, com: ScrapeCommittee):
+            if member.lower() == "no member data available":
+                return
+            # H/S members have (party/district) info
+            if "(" in member:
+                memberName = member.split("(", 1)[0].strip()
+                memberRole = member.rsplit(")", 1)[1].strip()
+                if memberRole:
+                    memberRole = memberRole.split(",", 1)[1].strip()
+                    com.add_member(name=memberName, role=memberRole)
+                else:
+                    com.add_member(name=memberName)
+            # Public members have names, role
+            elif "," in member:
+                memberName, memberRole = member.split(",", 1)
+                if memberRole:
+                    com.add_member(name=memberName.strip(), role=memberRole.strip())
+            else:
+                memberName = member.strip()
+                if not memberName:
+                    return
+                com.add_member(name=memberName)
+
+        # page header holds name and chamber
+        header = XPath("//*[@id=\"content\"]/div/section/h1").match_one(self.root).text.strip().rsplit(" ", 1)
+        name = header[0]
+        chamber = None
+        if "s" in header[1].lower():
+            chamber = "upper"
+        elif "h" in header[1].lower():
+            chamber = "lower"
+        elif "j" in header[1].lower():
+            chamber = "legislature"
+        else:
+            raise SkipItem("No chamber")
+
+        # determines if subcommittee
+        isSubcommittee = False
+        if "subcommittee" in name.lower():
+            isSubcommittee = True
+
+        # Conditionally create because altering parent/classification
+        # gives error
+        com: ScrapeCommittee()
+        if isSubcommittee:
+            com = ScrapeCommittee(
+                name=name,
+                chamber=chamber,
+                parent="Appropriations",
+                classification="subcommittee"
+            )
+        else:
+            com = ScrapeCommittee(
+                name=name,
+                chamber=chamber,
+                classification="committee"
+            )
+        com.add_source(self.source.url)
+        com.add_source("https://www.legis.iowa.gov/committees", "Committee list page")
+        # Get list of members contained in nested grid_12
+        # Translation hack in case they switch to all-caps
+        try:
+            membersList = XPath(
+                "//section/div/section[contains(@class, 'grid_12 alpha omega') and \
+                (contains(translate(., 'ABCDEFGHIJKLMNOPQRSTUVWXYZ','abcdefghijklmnopqrstuvwxyz'), 'house members') or \
+                contains(translate(., 'ABCDEFGHIJKLMNOPQRSTUVWXYZ','abcdefghijklmnopqrstuvwxyz'), 'senate members'))] //descendant::li"
+            ).match(self.root)
+        except:
```

Comment:
As the linting will tell you, python doesn't like "bare except"s. We need this to be _at least_ `except Exception:`.